src/components/organisms/RadioModal/RadioModal.tsx

```diff
@@ -34,15 +36,7 @@ export const RadioModal: React.FunctionComponent<RadioModalPropsType> = ({
         src={"/radio-icon-color.png"}
         alt="radio-icon"
       />
-      <input
-        type="range"
-        id="vol"
-        name="vol"
-        min="0"
-        max="100"
-        onChange={(ev) => setVolume(Number(ev.target.value))}
-        value={volume}
-      />
+      <VolumeControl volume={volume} onVolume={setVolume} name="vol" />
```

Comment:
is that correct that VolumeControl does not show anything 
![image](https://user-images.githubusercontent.com/63313732/130780810-2ee3f6eb-9ea1-4b9b-b7da-ad169d377f39.png)

previously it was this way:
![image](https://user-images.githubusercontent.com/63313732/130780868-c048b58d-8dba-41ba-b353-b1bfc5668a02.png)

Reply:
This one needs a prop flag. Good catch. I just don't have a reference venue to compare it before/after

Reply:
The Return of the Jed... Radio Slider @ eda678a6b658cc936def13246a4656c0b906b30d